src/helpers/bills.js

```diff
@@ -189,6 +192,47 @@ exports.formatAndCreateBills = async (groupByCustomerBills, credentials) => {
   );
 };
 
+exports.formatAndCreateBill = async (payload, credentials) => {
+  const { customer, date, billingItemList, netInclTaxes } = payload;
+  const { company } = credentials;
+
+  const billNumber = await exports.getBillNumber(date, company._id);
+  billNumber.seq += 1;
+  const number = exports.formatBillNumber(company.prefixNumber, billNumber.prefix, billNumber.seq);
+
+  const billBillingItemList = [];
+  for (const bi of billingItemList) {
+    const bddBillingItem = await BillingItem.findOne({ _id: bi.billingItem }, { vat: 1 }).lean();
```

Comment:
tu pourrais faire une appel global avec  un `{ _id: { $in` puis faire un find pour limiter le nombre d'appel a la bdd qu'en penses tu ?

Reply:
bien vu ! done